bitrise.yml

```diff
@@ -0,0 +1,95 @@
+---
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: react-native
+
+workflows:
+
+  primary:
+    description: |
+      Runs tests.
+
+      Next steps:
+      - Check out [Getting started with React Native apps](https://devcenter.bitrise.io/en/getting-started/getting-started-with-react-native-apps.html).
+    steps:
+    - activate-ssh-key@4: { }
+    - git-clone@6: { }
+    - yarn@0:
+        inputs:
+        - workdir: $WORKDIR
+        - command: install
+    - yarn@0:
+        inputs:
+        - workdir: $WORKDIR
+        - command: test
+    - deploy-to-bitrise-io@2: { }
+
+  deploy:
+    description: |
+      Tests, builds and deploys the app using *Deploy to bitrise.io* Step.
+
+      Next steps:
+      - Set up an [Apple service with API key](https://devcenter.bitrise.io/en/accounts/connecting-to-services/connecting-to-an-apple-service-with-api-key.html).
+      - Check out [Getting started with React Native apps](https://devcenter.bitrise.io/en/getting-started/getting-started-with-react-native-apps.html).
+    steps:
+    - activate-ssh-key@4: { }
+    - git-clone@6: { }
+    - yarn@0:
+        inputs:
+        - workdir: $WORKDIR
+        - command: install
+    - yarn@0:
+        inputs:
+        - workdir: $WORKDIR
+        - command: test
+    - install-missing-android-tools@3:
+        inputs:
+        - gradlew_path: $PROJECT_LOCATION/gradlew
+    - android-build@1:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $MODULE
+        - variant: $VARIANT
+    - cocoapods-install@2: { }
+    - xcode-archive@4:
+        inputs:
+        - project_path: $BITRISE_PROJECT_PATH
+        - scheme: $BITRISE_SCHEME
+        - distribution_method: $BITRISE_DISTRIBUTION_METHOD
+        - configuration: Release
+        - automatic_code_signing: api-key
+    - deploy-to-bitrise-io@2: { }
+
+trigger_map:
+- pull_request_source_branch: "*"
+  workflow: primary
+  pull_request_target_branch: main
+
+meta:
+  bitrise.io:
+    stack: osx-xcode-13.2.x
```

Comment:
Can we avoid hardcoding a stack here? I'm sure we'll forget to update this when new stacks are available, and one day builds will fail when the 13.2 stack is removed. What happens if we don't define a stack at all?

Reply:
Looks like the [iOS one is specifying a stack as well](https://github.com/bitrise-io/Bitrise-iOS-Cocoapods-Sample/blob/18fe2f5d9998930b3681cfd011f46ce624aba1b2/bitrise.yml#L12), so maybe it's ok? I'll check with the team.

Reply:
Whoops, my bad. Let me know if you find a quick and easy solution for this issue and I'll update the existing sample apps too

Reply:
haha, no worries. Wasn't pointing fingers, just wondering now if it effects the demo app builds. I'll find out and let you know. 👌

Reply:
This has been removed now 👍